internal/stats/metrics_recorder_list_test.go

```diff
@@ -223,7 +223,7 @@ func (s) TestMetricRecorderListPanic(t *testing.T) {
 	})
 	mrl := istats.NewMetricsRecorderList(nil)
 
-	want := "Length of labels passed to Record incorrect, got: 1, want: 2."
+	want := `Length of labels passed to Record incorrect for metric "simple counter".`
```

Comment:
It's not super important to check the string, especially in its entirety, if you don't want to keep adjusting this.

Reply:
Left it I like it.